viewer/.eslintrc.js

```diff
@@ -35,6 +35,10 @@ module.exports = {
     'react/prop-types': 'warn',
     'import/no-named-as-default': 'off',
     'react/static-property-placement': 'off',
+    'react/display-name': 'off',
+    '@typescript-eslint/no-var-requires': 'off',
     'no-unused-vars': 'warn',
+
+    'no-console': 'error', // we don't want to ship with console logs
```

Comment:
I'd set this to "warn" because setting it to"error" will prevent the application from building in the local Dev server. This is a productivity hit for developers. Unless there is some create-react-app way to configure it so it builds _locally_ even with lint errors, I think this should be "warn".